shared/README.md

```diff
@@ -136,26 +136,115 @@ You'll get various prompts about installing various tools. You should
 install 'Build Tools' and any missing platforms. However, _don't_
 update the Android Gradle Plugin to 3.0.1.
 
+Also see below for some messages you may encounter with Android Studio.
+
 Some instructions talk about the SDK Manager / AVD Manager. This is
-under the Tools > Android menu. You may have to wait for the Gradle to
+under the Tools > Android menu. You may have to wait for Gradle to
 sync before they appear.
 
 To run on the Android simulator:
 
 ```sh
 # Build the go keybaselib
-yarn run rn-gobuild-android
-# if this fails with something like 'xcrun: error: unable to lookup item 'Path' in SDK 'iphoneos' you might have a partial xcode install. try
-xcode-select --switch /Applications/Xcode.app/Contents/Developer
```

Comment:
i think we should leave this in -- i ran into it when just trying to build android stuff

Reply:
Oops, done. I had misread the comment. I reformatted it to be slightly clearer (it was scrolling off to the right)

Reply:
thanks!